web/assets/css/home.css

```
@@ -0,0 +1,195 @@
+}
+
+.tum-live-badge {
+    @apply rounded-full px-2 py-1
+}
+
+.tum-live-badge > i{
+    @apply mr-1
+}
+
+.tum-live-side-navigation{
+    @apply flex-col sticky top-0 overflow-y-scroll font-light text-3 shrink-0 divide-y dark:divide-gray-800
+}
+
+.tum-live-side-navigation > .tum-live-side-navigation-group {
+    @apply px-5
+}
+
+.tum-live-side-navigation-group{
+    @apply grid py-4 content-baseline
+}
+
+.tum-live-side-navigation-group > header {
+    @apply flex items-center px-3 pb-2 font-semibold
+}
+
+.tum-live-side-navigation-group > header > i {
+    @apply text-xs mr-2
+}
+
+.tum-live-side-navigation-group-item {
+    @apply text-sm rounded-lg text-left px-3 py-2
+}
+
+.tum-live-side-navigation-group-item ~ .hover {
+    @apply hover:bg-gray-100 dark:hover:bg-gray-800
+}
+
+.tum-live-side-navigation-group-item > i {
+    @apply text-xs mr-2
+}
+
+.tum-live-thumbnail {
+    @apply relative bg-gray-100 dark:bg-gray-800 rounded-lg mb-2 bg-cover transition-all duration-500 hover:shadow-xl dark:shadow-gray-900/75;
+}
+
+.tum-live-thumbnail-progress {
+    @apply absolute bottom-0 w-full
+}
+
+.tum-live-thumbnail-progress > div {
+    @apply relative w-full h-2 bg-gray-500/50 rounded-b-lg
+}
+
+.tum-live-thumbnail-progress > div > span {
+    @apply absolute left-0 tum-live-bg-primary h-2 rounded-bl-lg
+}
+
+.tum-live-content-grid {
+    @apply grid gap-y-8 pb-4 h-full
+}
+
+.tum-live-content-grid-item > h3{
+    @apply font-bold px-3
+}
+
+.tum-live-stream .course {
+    @apply block text-5 hover:bg-gray-50 dark:hover:bg-gray-800 rounded
+}
+
+.tum-live-stream .title {
+    @apply block font-medium hover:underline
+}
+
+.tum-live-stream .date {
+    @apply flex items-center font-light;
+}
+
+.tum-live-menu {
+    @apply tum-live-bg shadow-lg rounded-lg h-fit border dark:border-gray-800 text-sm text-3 dark:shadow-gray-900/75
+}
+
+
+.tum-live-menu > header {
+    @apply py-2 px-4 border-b dark:border-gray-800
+}
+
+.tum-live-menu > header:has(.close) {
+    @apply flex justify-between items-center
+}
+
+.tum-live-menu > header .close{
+    @apply ml-auto
+}
+
+.tum-live-course-list > header {
+    @apply flex items-center justify-between mb-8
+}
+
+.tum-live-course-list > header > h1 {
+    @apply flex items-center font-bold px-3
+}
+
+.tum-live-course-list > header > button {
+    @apply tum-live-button tum-live-button-secondary
+}
+
+.tum-live-course-list-item {
+    @apply hover:bg-gray-100 dark:hover:bg-gray-800 rounded-lg px-3 py-2
+}
+
+.tum-live-course-list-item .title {
+    @apply block text-3 lg:text-base text-sm font-semibold hover:underline
+}
+
+.tum-live-course-list-item .links {
+    @apply flex flex-col text-sm text-5 font-light lg:flex-row lg:items-center
+}
+
+.tum-live-course-list-item .links > *:not(:first-child){
+    @apply pt-1 lg:pt-0 lg:pl-2
+}
+
+.tum-live-footer {
+    @apply bg-gray-100 dark:bg-secondary-light py-4 px-8;
+}
+
+.tum-live-footer a {
+    @apply text-sm font-light text-5 md:py-0 py-1
+}
```

Not a strong opinion, but I think this could be simplified this way

```suggestion
footer {
    @apply bg-gray-100 dark:bg-secondary-light py-4 px-8;
}

footer a {
    @apply text-sm font-light text-5 md:py-0 py-1
}
```

---

The problem with `footer` alone is that the `<footer>` tag can be used multiple times for different purposes. (For example as page-footer or menu-footer) Therefore introducing a class makes sense to me.